src/V1/Actor/Writer.service.yml

```diff
@@ -1,7 +1,11 @@
+parameters:
+  env(Neighborhoods_Buphalo_V1_Actor_WriterInterface__SourceOverrideBehavior): !php/const \Neighborhoods\Buphalo\V1\Actor\WriterInterface::SOURCE_OVERRIDE_BEHAVIOR_DEFAULT
```

Comment:
I take it this ensures the default is always being used and line 3 then overrides the default if a variable is provided?